lib/galaxy/datatypes/tabular.py

```diff
@@ -53,26 +53,22 @@ def displayable( self, dataset ):
         except:
             return False
 
-    def get_chunk(self, trans, dataset, chunk):
-        ck_index = int(chunk)
-        f = open(dataset.file_name)
-        f.seek(ck_index * trans.app.config.display_chunk_size)
-        # If we aren't at the start of the file, seek to next newline.  Do this better eventually.
-        if f.tell() != 0:
+    def get_chunk(self, trans, dataset, offset=0, ck_size=0):
```

Comment:
I'd prefer `None` as default for `ck_size` (which more clearly represents not defined instead of 0).